scripts/README.md

```diff
@@ -0,0 +1,14 @@
+Instructions for esmDependencyCheck.js
+
+This script was written with the intent to have a way to quickly check if any of our dependencies are ESM (instead of CommonJS), and with that knowledge make the decision if and when to upgrade or replace them based on the fact changing to ESM can be difficult. It may become less difficult in the future as the ecosystem matures.
+
+To run it use 'yarn run esmDependencyCheck'.
+
+In order to use this script, you need to put your GIT_TOKEN in your .env file. 
```

Comment:
It would be easier to add `GIT_TOKEN` as an environment variable, because the .env file gets overwritten every time you do a `yarn dev`. 

You have 2 options:

1. add GIT_TOKEN to your bash profile (in the same way we do the BFF_PATH) - (we may also need to provide a link to instructions on how to get a GIT_TOKEN as not everyone will have it set up on their GitHub account)
2. Add GIT_TOKEN on the command line when you run the command e.g. `GIT_TOKEN=XXX yarn run esmDependencyCheck`

This then removes the need for the dotenv code in the script.

Reply:
I have done option 1 for myself and provided both options within the README.